heft-plugins/heft-sass-plugin/src/index.ts

```diff
@@ -1,16 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license.
```

Comment:
Didn't get cleaned up in an earlier PR?

Reply:
Not for all plugins, no. That feedback came after this file was merged IIRC. It's cleaned up now :)